execution-environments/2.15-with-ansible8/execution-environment.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
The directory name doesn't look good. This collection does **not** contain Ansible, it only contains a few collections that are part of Ansible.